src/chromatix/functional/samples.py

```diff
@@ -158,8 +159,13 @@ def multislice_thick_sample(
     # Propagate field backwards to the middle (or chosen distance) of the stack
     if reverse_propagate_distance is None:
         reverse_propagate_distance = thickness_per_slice * absorption_stack.shape[0] / 2
-    field = exact_propagate(
-        field, z=-reverse_propagate_distance, n=n, kykx=kykx, N_pad=0
+    field = asm_propagate(
+        field,
+        z=-reverse_propagate_distance,
+        n=n,
+        kykx=kykx,
+        N_pad=0,
+        remove_evanescent=True,
```

Comment:
This was initially added with the exact propagator (now `remove_evanescent=True`), but I think we should actually be leaving the evanescent waves in. This is what I've done when trying to match original implementations in MATLAB by Shwetadwip and his lab. And I think we also leave them in for the fluorescence scattering model by Cedric, where we found that this actually seemed to matter.

Reply:
Ok I'll remove the remove 😉 

Should we also keep evanescent waves in Line 151 just above?

Reply:
Yup, let's keep evanescent waves everywhere that we use ASM by default, since that seems to be the more popular (?) choice.